src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -246,6 +246,12 @@ type Service interface {
 
 	// Only safe to be called one time once the service has started.
 	SetDatabase(db storage.Database) error
+
+	// SetSupportsSnappyCompression sets whether snappy compression is enabled.
+	SetSupportsSnappyCompression(value bool)
+
+	// SupportsSnappyCompression returns whether snappy compression is enabled.
+	SupportsSnappyCompression() bool
```

Comment:
Would these be better as "SupportedCompressions" and then it can yield all of the formats it supports? Probably fine to relegate to future work, although may be hard to change rpc interfaces

Reply:
Agreed.